internal/k8s/k8s.go

```diff
@@ -68,17 +72,18 @@ const (
 // Config specifies the configuration of the Kubernetes
 // client/watcher.
 type Config struct {
-	ProcessName   string
-	ConfigMapName string
-	ConfigMapNS   string
-	NodeName      string
-	MetricsHost   string
-	MetricsPort   int
-	ReadEndpoints bool
-	Logger        log.Logger
-	Kubeconfig    string
-
-	ServiceChanged func(log.Logger, string, *v1.Service, *v1.Endpoints) SyncState
+	ProcessName        string
+	ConfigMapName      string
+	ConfigMapNS        string
+	NodeName           string
+	MetricsHost        string
+	MetricsPort        int
+	ReadEndpoints      bool
+	ReadEndpointSlices bool
```

Comment:
At the moment, this isn't used. Do we need it? Can you think of a good reason to disable this if EndpointSlices were enabled on the cluster?

Reply:
Nope, this is a leftover of a previous version where I intended this to be configurable. Now it's driven by the same ReadEndpoints variable. I'll remove this